indexer/packages/v4-protos/src/codegen/dydxprotocol/indexer/off_chain_updates/off_chain_updates.ts

```diff
@@ -220,12 +221,18 @@ export function orderRemoveV1_OrderRemovalStatusToJSON(object: OrderRemoveV1_Ord
 export interface OrderPlaceV1 {
   order?: IndexerOrder;
   placementStatus: OrderPlaceV1_OrderPlacementStatus;
+  /** The timestamp of the order placement. */
+
+  timeStamp?: Date;
```

Comment:
```suggestion
  timeStamp?: Date; // The timestamp of the order placement.
```
nit: and for all the fields below